go/adbc/driver/snowflake/driver_test.go

```diff
@@ -677,87 +674,88 @@ func (suite *SnowflakeTests) TestUseHighPrecision() {
 	suite.Equal(9876543210.99, rec.Column(1).(*array.Float64).Value(1))
 }
 
-func TestUnencryptedJwtAuthenticationUsingString(t *testing.T) {
-	drv := gosnowflake.SnowflakeDriver{}
-
+func TestJwtAuthenticationUnencryptedValue(t *testing.T) {
+	// test doesn't participate in SnowflakeTests because of the
+	// JWT auth has a different behavior
 	uri, ok := os.LookupEnv("SNOWFLAKE_URI")
-
 	if !ok {
-		panic("Cannot find the `SNOWFLAKE_URI` value")
+		t.Skip("Cannot find the `SNOWFLAKE_URI` value")
 	}
 
-	cfg, _ := gosnowflake.ParseDSN(uri)
-	cfg.Authenticator = gosnowflake.AuthTypeJwt
-
 	keyValue, ok := os.LookupEnv("SNOWFLAKE_TEST_PKCS8_VALUE")
```

Comment:
does this need to get added as a secret on the repo? or do we have a hardcoded test value to use?

Reply:
It's not as straightforward as adding a secret. We _could_ but it also requires adding the public key to the user's Snowflake account, so we'd need something like what is in https://github.com/apache/arrow-adbc/pull/1199 to test with.

Reply:
I'd be fine with using something like that to test, or since I have access to the account we use for the CI here via the secrets I can add a key to account and add the secrets.

It's probably easier to just use the same setup as in #1199 though

Reply:
I emailed Matt on the side to sort it out. We are going to add secrets to the repo.